helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -41,3 +29,36 @@ tgi:
     periodSeconds: 5
     timeoutSeconds: 1
     failureThreshold: 120
+
+# Reranking: second largest bottleneck when reranking is in use
+# (i.e. query context docs have been uploaded with data-prep)
+teirerank:
+  accelDevice: "gaudi"
+  image:
+    repository: opea/tei-gaudi
+    tag: "1.0"
```

Comment:
For all opea images, we use "latest" tag in the main branch, and change it later in the release-rc branch. Please use "latest" here.

Reply:
done